tests/test_config.py

```diff
@@ -212,3 +214,30 @@ def test_ini_config_bash():
     cfg.update({'dressing': ['ranch', 'italian']})
     expected['dressing'] = ['ranch', 'italian']
     assert cfg == expected
+
+def test_transform_config():
+    '''Test that transforms work as intended.
```

Comment:
This might be clearer as "...transforms config objects to objects of other config subclasses". It might not be, but I think we should attempt to give the doc block more meaning here to tell folks what the point of the test is.